plugins/ggdm30.js

```diff
@@ -234,100 +234,53 @@ ggdm30 = {
 
         if (attrList != undefined)
         {
-            // The code is duplicated but it is quicker than doing the "if" on each iteration
```

Comment:
This has been removed in all the translations since I am the only one using this option.